sphinx_needs/css/blank/styles.css

```diff
@@ -90,3 +90,6 @@ table.needs_style_blue_bar {
     background: linear-gradient(to right, transparent 90%, #0fbcf9 10%);
 }
 
+.collapse_is_visible {
```

Comment:
I just compared this class with below jQuery selector. Would the selector here still work if prefixed like this
```
span.needs.collapse_is_visible {
```
? If yes, I'd appreciate to use a more detailed selector, just to be sure it does not clash with user-space CSS.

Reply:
Yes it will.

Reply:
Ok, thanks for the update.
Then I'd vote for
```
span.needs.collapse_is_hidden {
    display: none;
} 
```
The selector is as specific as possible.
A class that makes something disappear should have something like 'hidden' in its name.
Any thoughts?

Reply:
Yeah, collapse-is-hidden is ok too